Source/Core/DolphinQt/Config/PropertiesDialog.cpp

```diff
@@ -32,6 +32,7 @@ PropertiesDialog::PropertiesDialog(QWidget* parent, const UICommon::GameFile& ga
                           QString::fromStdString(game.GetGameID()),
                           QString::fromStdString(game.GetLongName())));
   setWindowFlags(windowFlags() & ~Qt::WindowContextHelpButtonHint);
+  setAttribute(Qt::WA_DeleteOnClose, true);
```

Comment:
This should probably have a comment accompanying it explaining why this behavior is needed.

Reply:
Alternatively, maybe the code would be clear enough on its own without needing a comment if we move this line to where the `PropertiesDialog` gets allocated? That is how every other usage of `Qt::WA_DeleteOnClose` in Dolphin seems to be done.

Reply:
Added the comment, but I think what JosJuice mentions is better; I'll do that now.

Reply:
Changed the comment and moved it to the allocation point.  (For reference, the previous comment was `The properties dialog is not re-used, and maintains a lock on the game file until it's deleted, preventing the file from being moved or deleted.`)